ghost/admin/app/components/gh-resource-select.js

```diff
@@ -146,4 +153,38 @@ export default class GhResourceSelect extends Component {
 
         this._options = options;
     }
+
+    @task({restartable: true})
+    *fetchOptionsForSearchTask(searchTerm) {
+        // Debounce
+        yield timeout(DEBOUNCE_MS);
+
+        const options = yield [];
+
+        if (this.args.type === 'email') {
+            const posts = yield this.store.query('post', {filter: '(status:published,status:sent)+newsletter_id:-null+title:~\'' + searchTerm.replace('\'', '\\\'') + '\'', limit: '10', fields: 'id,title'});
+            options.push(...posts.map(mapResource));
+            return;
+        }
+
+        const posts = yield this.store.query('post', {filter: 'status:published+title:~\'' + searchTerm.replace('\'', '\\\'') + '\'', limit: '10', fields: 'id,title'});
+        const pages = yield this.store.query('page', {filter: 'status:published+title:~\'' + searchTerm.replace('\'', '\\\'') + '\'', limit: '10', fields: 'id,title'});
+
+        function mapResource(resource) {
+            return {
+                id: resource.id,
+                title: resource.title
+            };
+        }
```

Comment:
May want to extract `mapResource` out of the task, to avoid redefining it every time `fetchOptionsForSearchTask` is invoked? Also used by the `fetchOptionsTask` task above

Reply:
Ah did that, forgot to push that change 🙈